recreateBenchmark.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+set -eo pipefail
+
+# The purpose of this script is to make it easier for developers to recreate benchmarks.
+# As input the benchmark name is expected, which is used for the docker image tag and k8 namespace.
+# This script does the following:
+#
+# 1. Build the current branch
+# 2. Build the docker dev image
+# 3. Publish the docker image
+# 4. Redeploys the benchmark
+
+# Contains OS specific sed function
+source benchmarks/setup/utils.sh
+
+function printUsageAndExit {
+  printf "\nUsage ./createBenchmark <benchmark-name>\n"
+  exit 1
+}
+
+if [[ -z $1 ]]
+then
+  echo "<benchmark-name> not provided"
+  printUsageAndExit
+fi
+benchmark=$1
+
+# DNS Label regex, see https://regex101.com/r/vjsrEy/2
+if [[ ! $benchmark =~ ^[a-z0-9]([-a-z0-9]{0,61}[a-z0-9])?$ ]]; then
+  echo "<benchmark-name> '$benchmark' not a valid DNS label"
+  echo "See https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names"
+  printUsageAndExit
+fi
+
+set +e
+ns=$(kubectl get namespace "$benchmark")
+set -e
+if [[ ! "$ns" ]]; then
+  echo "Perhaps you meant to use ./createBenchmark.sh instead. Or consider switching your k8s context"
+  exit 1
+fi
+
+# Check if docker daemon is running
+if ! docker info >/dev/null 2>&1; then
+    echo "Docker daemon does not seem to be running, make sure it's running and retry"
+    exit 1
+fi
+
+set -x
+
+mvn clean install -DskipTests -DskipChecks -T1C
+
+docker build --build-arg DISTBALL=dist/target/camunda-cloud-zeebe-*.tar.gz --build-arg APP_ENV=dev -t "gcr.io/zeebe-io/zeebe:$benchmark" .
+docker push "gcr.io/zeebe-io/zeebe:$benchmark"
+
+cd "benchmarks/setup/$benchmark"
+
+make update
```

Comment:
I guess this is not needed as we are not changing helm config. Also this has no effect.

Reply:
I've removed it 👍